README.md

```diff
@@ -6,12 +6,13 @@ GenAIInfra is the containerization and cloud native suite for OPEA, including ar
 
 The GenAIInfra repository is organized under four main directories, which include artifacts for OPEA deploying:
 
-| Directory           | Purpose                                                                                                                     |
-| ------------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| `helm-charts`       | Helm charts for deploying [GenAIExamples](https://github.com/opea-project/GenAIExamples) on Kubernetes.                     |
-| `kubernetes-addons` | Deploy Kubernetes add-ons for OPEA.                                                                                         |
-| `manifests`         | Manifests for deploying [GenAIExamples](https://github.com/opea-project/GenAIExamples) on Kubernetes and on Docker Compose. |
-| `scripts`           | Scripts for testing, tools for facilitate OPEA deployment, and etc.                                                         |
+| Directory                 | Purpose                                                                                                                                      |
+| ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| `microservices-connector` | GenAI Microservices Connector(GMC) for deploying and adjusting [GenAIExamples](https://github.com/opea-project/GenAIExamples) on Kubernetes. |
```

Comment:
How about use `orchestrating` for describing `microservices-connector`? 
Suggest that: 
```
GenAI Microservices Connector(GMC) for orchestrating AI workloads and deploying [GenAIExamples](https://github.com/opea-project/GenAIExamples) on Kubernetes.
```

Reply:
But GMC is more than orchestrator -- that would just launch a microservice on a node that best suits it or is free type thing. GMC also connects things and makes pipelines, watches for changes in pipeline definition in a declarative manner and adapts their launching and chaining.  It also supports way more than the GenAIExamples, honoring any yaml file that follows mega language.

How about "
GenAI Microservices Connector (GMC) supports the launching, monitoring, and updating of GenAI microservice chains, such as those in [GenAIExamples](https://github.com/opea-project/GenAIExamples), on Kubernetes. It essentially supports a Kubernetes Custom Resource Definition for GenAI chains/pipelines that may be comprised of sequential, conditional, and parallel steps."